src/server/app/resources/services/resources.server.service.js

```diff
@@ -109,11 +91,11 @@ module.exports = function() {
 							'owner._id': user._id
 						}];
 
-						return doSearch(query, sortParams, page, limit);
+						return Resource.pagingSearch(query, sortParams, page, limit);
```

Comment:
Done! I also added some coverage for the `resources` service. Thanks for catching that!